interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
And I also agree that we should update all of our frontend toolchain, but that it would make sense to do it once we have a sturdy Docker environment making builds consistently.